config.go

```diff
@@ -257,12 +263,30 @@ func SetupDaemonConfig(logger *logrus.Logger, configFile string) (DaemonConfig,
 		}
 	}
 
+	// Log config
+	setter.SetDefault(&logConfig.LogFormat, os.Getenv("GUBER_LOG_FORMAT"))
+	if logConfig.LogFormat != "" {
+		switch logConfig.LogFormat {
+		case "json":
+			logger.SetFormatter(&logrus.JSONFormatter{})
+		case "text":
+			logger.SetFormatter(&logrus.TextFormatter{})
+		default:
+			return conf, errors.New("GUBER_LOG_FORMAT is invalid; expected value is either json or text")
+		}
+	}
+
 	setter.SetDefault(&DebugEnabled, getEnvBool(log, "GUBER_DEBUG"))
 	if DebugEnabled {
 		logger.SetLevel(logrus.DebugLevel)
 		log.Debug("Debug enabled")
 	}
 
+	setter.SetDefault(&logConfig.LogLevel, uint32(getEnvInteger(log, "GUBER_LOG_LEVEL")))
+	if !DebugEnabled {
```

Comment:
This logic is tied to line 280.  It would make sense this be together in the same `if`/`else` condition.

Reply:
See code example in previous comment.

Reply:
Oops, I just missed this comment @Baliedge, let me update it.